miso-web/src/main/webapp/scripts/box_visualization.js

```diff
@@ -172,9 +172,9 @@
 
       var tBody = jQuery('<tBody>');
       var tRow = jQuery('<tr>');
-      var emptyCornerCell = jQuery('<th>');
-      emptyCornerCell.css('width', '30px');
-      emptyCornerCell.css('height', '30px');
+      var emptyCornerCell = jQuery('<th>').text('#');
+      emptyCornerCell.addClass('boxVisColHeader boxVisRowHeader');
+      emptyCornerCell.click(self.selectAll);
```

Comment:
Can this be made more obvious how to access the "Select All" functionality? I only knew where to click because I came back here and read the code... maybe a "Select All" button next to the "Select Odd Columns" and "Select Even Columns" buttons?

Reply:
This implementation should be familiar from standard spreadsheet software. I just added the '#' to make it a bit more obvious. I'd rather not add more buttons to the page if it can be avoided. How do you feel about adding "Click the '#' at the top left to select all." to the help at the top instead?